decidim-admin/app/views/decidim/admin/scope_types/index.html.erb

```diff
@@ -0,0 +1,40 @@
+<div class="card">
+  <div class="card-divider">
+    <h2 class="card-title"><%= t "decidim.admin.titles.scope_types" %> <%= link_to t("actions.add", scope: "decidim.admin"), ['new', 'scope_type'], class: 'button tiny button--title' %></h2>
+  </div>
+  <div class="card-section">
+    <div class="table-scroll">
+        <table class="table-list">
+        <thead>
+          <tr>
+            <th><%= t("models.scope_type.fields.name", scope: "decidim.admin") %></th>
+            <th><%= t("models.scope_type.fields.plural", scope: "decidim.admin") %></th>
+            <th></th>
+          </tr>
+        </thead>
+        <tbody>
+          <% @scope_types.each do |scope_type| %>
```

Comment:
I think @oriolgual means the `collection` method defined in the controller. I think this is used in the participatory process controller as some trick to render heterogeneous collections (process + process_groups) transparently. But in this case I think we don't need that. I would rename the `collection` method to `scope_types`, expose it as a helper and use that.